cmd/apps/nginx_app.go

```diff
@@ -34,51 +36,20 @@ flag and the ingress-nginx docs for more info`,
 	nginx.Flags().StringArray("set", []string{}, "Use custom flags or override existing flags \n(example --set=image=org/repo:tag)")
 
 	nginx.RunE = func(command *cobra.Command, args []string) error {
-		kubeConfigPath := config.GetDefaultKubeconfig()
-		wait, _ := command.Flags().GetBool("wait")
```

Comment:
this flag has been removed and is still a valid option on install, can it be added back in and passed to options with .WithWait()

Reply:
Thank you for pointing this. I have added it.